shared/macros-bash.jinja

```diff
@@ -444,34 +444,34 @@ printf '%s\n' "{{{ message | replace('"', '\\"') }}}" >&2
 # separator_regex: regular expression that describes the separator and surrounding whitespace
 # prefix_regex: regular expression describing allowed leading characters at each line
 #}}
-{{%- macro set_config_file(path, parameter, value, create, insert_after, insert_before, insensitive=true, separator=" ", separator_regex="\s\+", prefix_regex="^\s*") -%}}
+{{%- macro set_config_file(path, parameter, value, create, insert_after, insert_before, insensitive=true, separator=" ", separator_regex="\s\+", prefix_regex="^\s*", sed_path_separator="/") -%}}
     {{%- set line_regex = prefix_regex+parameter+separator_regex -%}}
     {{%- set new_line = parameter+separator+value -%}}
 if [ -e "{{{ path }}}" ] ; then
-    {{{ lineinfile_absent(path, line_regex, insensitive) | indent(4) }}}
+    {{{ lineinfile_absent(path, line_regex, insensitive, sed_path_separator) | indent(4) }}}
```

Comment:
Since `sed_path_separator` is a named parameter, I'd pass it here as a named parameter too, just in case more positional parameters are added later.